### ISSUE TITLE
Add a static demo with per-PR review

### ISSUE THREAD

**Issue by bollwyvl** [jupyterlite/xeus-sqlite-kernel#7]
### Problem

There's no place to try out this kernel yet.

Aside from off `main`, as new features are added, it would be nice to have something that reviewers can try out.

### Suggested Improvement

- [ ] add a dev-focused binder with `jupyter-server-proxy`
- [ ] get set up for readthedocs (doesn't even require the mamb flag anymore to get a self-consistent environment)

**Comment by jtpio** [comment]
Actually there is one in the https://github.com/jupyterlite/demo repo: https://github.com/jupyterlite/demo/blob/main/content/xeus-sqlite/simple-operations.ipynb

Alongside other xeus-based kernels.

**Comment by bollwyvl** [comment]
Right, that's a starting point, but stuff has to hit main to show up there,
presumably...

**Comment by psychemedia** [comment]
I also note that this repo builds on [`jupyter-xeus/xeus-sqlite`](https://github.com/jupyter-xeus/xeus-sqlite/), and that repo has upstream fixes such as https://github.com/jupyter-xeus/xeus-sqlite/pull/119 that will need to be tracked here and in downstream demo environments if folk want to demo `latest`.

**Comment by jtpio** [comment]
With the ongoing refactoring of the xeus-based lite kernels, it's also possible the lite assets will eventually be developed and shipped from the main [jupyter-xeus/xeus-sqlite](https://github.com/jupyter-xeus/xeus-sqlite/) repo directly.

Making https://github.com/jupyterlite/xeus-sqlite-kernel not strictly necessary.
